downloads/swan-lake-release-notes/swan-lake-alpha3/RELEASE_NOTE.md

```diff
@@ -2,41 +2,928 @@
+
+    isolated remote function onConnect(tcp:Caller caller) returns tcp:ConnectionService {
+        io:println("Client connected to secureEchoServer: ", caller.remotePort);
+        return new EchoService(caller);
+    }
+}
+
+service class EchoService {
+  
+    remote function onBytes(readonly & byte[] data) returns (readonly & byte[])|tcp:Error? {
+        io:println("Echo: ", 'string:fromBytes(data));
+        return data;
+    }
+}
+```
+
+- Included a `tcp:Caller` as an optional parameter in the `onBytes()` method.
+```ballerina
+service class EchoService {
+  
+    remote function onBytes(tcp:Caller caller, readonly & byte[] data) returns (readonly & byte[])|tcp:Error? {
+        io:println("Echo: ", 'string:fromBytes(data));
+        check caller->writeBytes(data);
+    }
+}
+```
+
+- Renamed `tcp:ListenerConfig` and `tcp:ClientConfig` to `tcp:ListenerConfiguration` and `tcp:ClientConfiguration`
+
+##### UDP Package Updates
+
+- Renamed `udp:ListenerConfig` and `udp:ClientConfig` to `udp:ListenerConfiguration` and `udp:ClientConfiguration`
+
+##### Kafka Package Updates
+
+- Renamed the `sendProducerRecord` function in the client object `Producer` to `send`.
+
+- Renamed the `flushRecords` function in the client object `Producer` to `’flush`.
+
+- Replaced the `kafka:ConsumerError` and `kafka:ProducerError` with `kafka:Error`.
+
+##### NATS Package Updates
+
+- Renamed the `ConnectionConfig` record to `ConnectionConfiguration`. 
+
+- Included `url` as a field in the `ConnectionConfiguration` record. 
+
+- Changed the `ConnectionConfiguration` in the client and listener init functions to an included record parameter. This allows the record field values to be passed as named parameters. 
+
+##### STAN Package Updates
+
+- Renamed the `StreamingConfig` record to `StreamingConfiguration`. 
+
+- Included the `url` as a field in the `StreamingConfiguration` record. 
+
+- Changed the `StreamingConfiguration` in the client and listener init functions to an included record parameter. This allows the record field values to be passed as named parameters. 
+
+##### RabbitMQ Package Updates
+
+- Renamed the `ConnectionConfig` record to `ConnectionConfiguration`. 
+
+###### Common Standard Library Updates
+
+- All the timeout configurations are converted to accept decimal values and the time unit is in seconds.
+
+##### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha3](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Swan+Lake+Alpha3%22).
+
+#### Code to Cloud
+
+- Removed Docker annotation support.
+
+##### Bug Fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake Alpha3 of the repositories below.
+
+- [C2C](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+Alpha3%22) 
```

Comment:
```suggestion
To view bug fixes, see the [GitHub milestone for Swan Lake Alpha3](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+Alpha3%22).
```

Reply:
@MaryamZi Based on the above comment, `[GitHub milestone for Swan Lake Alpha3]` should point here
https://github.com/ballerina-platform/ballerina-lang/milestone/105?closed=1

The link in your comment point to points only `c2c` bug fixes. In that case, we have to explicitly say for c2c bug fixes as I was earlier. WDYT?

Reply:
Fixed